Source/UnitTests/GTMSessionFetcherUtilityTest.m

```diff
@@ -120,6 +120,10 @@ - (void)testGTMFetcherSystemVersionString {
 }
 
 - (void)testGTMFetcherApplicationIdentifier {
+// Swift Packages do not have the notion of bundles,
+// and the actual identifier might change from release to release,
+// or wether was built using Xcode or swift command.
```

Comment:
How is the bundle id changing? it would be nice to still get test coverage for these things.

Reply:
When running the tests using `swift test` command, the bundle identifier will be **proc_xctest**. And if the same test is run under Xcode it will be **GTMSessionFetcherCoreTests**. it might be possible to change the bundle by passing some compiler flags, but this approach is hacky and might break in the future.